vue/components/actions/actions_bootstrap.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div v-if="actionList != null && actionList.length > 0" class="dynamicforms-actioncontrol">
+    <button
+      v-for="action in actionList"
+      :key="action.name + action.icon"
+      class="btn btn-info btn-sm"
+    >
+      <IonIcon :class="getDisplayStyle(action).showIcon" class="action-icon" :name="action.icon"/>
+      <span :class="getDisplayStyle(action).showMargin" style="width: .5rem"/>
+      <span :class="getDisplayStyle(action).showLabel">{{ action.label }}</span>
+    </button>
+  </div>
+</template>
+
+<script>
+import IonIcon from 'vue-ionicon';
+
+import Actions from './actions';
+
+export default {
+  name: 'BootstrapActions',
+  components: { IonIcon },
+  mixins: [Actions],
+  methods: {
+    getDisplayStyle(action) {
+      // TODO: Bootstrap button doesn't have button type like "text" in vuetify...
+      //  so we currently don't support it in bootstrap.
+      let res = this.displayStyle[action];
+      const iconClasses = {};
+      const labelClasses = {};
+
+      const iconClassesList = [];
+      const labelClassesList = [];
+      const marginClassesList = [];
+
+      if (res == null) {
+        res = { iconClass: '', labelClass: '' };
+        if (action.displayStyle) {
+          // We find display values for all sizes... if they are set. And than for specific sizes xs, sm, ..., xl
+          // Than we look from smallest to largest size, and set classes.
+          // If class is not defined for size we take one from previous size.
+          this.checkStyle(action.displayStyle, 'xs', iconClasses, labelClasses);
+          this.checkStyle(action.displayStyle.xs, 'xs', iconClasses, labelClasses);
+          this.checkStyle(action.displayStyle.sm, 'sm', iconClasses, labelClasses);
+          this.checkStyle(action.displayStyle.md, 'md', iconClasses, labelClasses);
+          this.checkStyle(action.displayStyle.lg, 'lg', iconClasses, labelClasses);
+          this.checkStyle(action.displayStyle.xl, 'xl', iconClasses, labelClasses);
+        }
+        this.checkStyle({ showLabel: true, showIcon: true }, 'def', iconClasses, labelClasses);
+
+        let iconClass = iconClasses.def;
```

Comment:
vsa ta koda je po mojem nepotrebna, ko enkrat uporabiš spodaj predlagani checkstyle.
breakpoints array lahko deklariraš izven komponente kot konstanto, oz. napišeš še eno `checkStyles` funkcijo, ki gre čez vse breakpointe za posamičen atribut in samo vrne CSS classes, kot jih rabiš za posamičen element actiona.

Funkcija mora samo dodajati rezultat checkStyle funkcije v končni string. npr:

`breakpoints.reduce((result, breakpoint) => result + checkStyle(this.displayStyle, 'asIcon', breakpoint), '')`